tools/apiview/emitters/cadl-apiview/cspell.yaml

```diff
@@ -0,0 +1,67 @@
+version: "0.2"
+language: en
+allowCompoundWords: true
+dictionaries:
+  - node
+  - typescript
+words:
+  - autorest
+  - azsdkengsys
+  - azurecr
+  - blockful
+  - blockless
+  - cadl
+  - Contoso
+  - CRUDL
+  - devdriven
+  - dogfood
+  - eastus
+  - esbuild
+  - globby
+  - Hdvcmxk
+  - inmemory
+  - instanceid
+  - intrinsics
+  - ints
+  - jsyaml
+  - msbuild
+  - MSRC
+  - munge
+  - mylib
+  - nostdlib
+  - oapi
+  - oneof
+  - onig
+  - onigasm
+  - onwarn
+  - openapi
+  - openapiv
+  - picocolors
+  - pnpm
+  - proto
+  - protobuf
+  - protoc
+  - regen
+  - rpaas
+  - rushx
+  - safeint
+  - strs
+  - TSES
+  - Uhoh
```

Comment:
I have no problem adding this, I'd just like to understand what the need for `Uhoh` is :)

Reply:
this is the file from `Microsoft/cadl`, it should maybe be cleaned of unused words

Reply:
> have no problem adding this, I'd just like to understand what the need for Uhoh is :)

@johanste have you _used_ APIView?